src/main/java/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -255,31 +256,18 @@ public synchronized void put(String key, Entry entry) {
     /** Removes the specified key from the cache if it exists. */
     @Override
     public synchronized void remove(String key) {
-        boolean deleted = getFileForKey(key).delete();
+        boolean deleted = DiskBasedCacheUtility.getFileForKey(key, mRootDirectorySupplier).delete();
         removeEntry(key);
```

Comment:
Can you use the new DiskBasedCacheUtility#removeEntry function here?

Reply:
Oh yeah. It looks like I didn't make those changes for DiskBasedCache on this branch.

Reply:
We should hold off reviewing this PR until the other PR on which it is based is merged?

Reply:
Yeah, I think that's smart